src/scripts.js

```diff
@@ -24,12 +24,16 @@ let weeklyStepCount = document.querySelector("#weeklyStepCount");
 let ouncesInput = document.querySelector("#ouncesInput");
 let form = document.querySelector("#form");
 let newEntry = document.querySelector('#newEntry');
-let calender = document.querySelector('#calender');
-
+let calendar = document.querySelector('#calendar');
+let calendar2 = document.querySelector('#calendar2');
+let activityNote = document.querySelector('#activityNotes')
+let activityInput = document.querySelector('#activityInput')
+let userNoteBtn = document.querySelector('#userNotes')
 
 let date = new Date();
 let currentDate = date.getFullYear() + "/" + ("0" + (date.getMonth()+1)).slice(-2) + "/"+ ("0" + date.getDate()).slice(-2);
 let newUser, users, hydration, sleep, activity;
+let activityNotes = [];
```

Comment:
This was a great idea to make an empty array and pushing the users data to it for us to iterate over!